hacker/admin.py

```diff
@@ -1,14 +1,39 @@
 from django.contrib import admin
 from .models import Hacker, Application, Confirmation, Team
 from django import forms
+import datetime
 
 
+''' `Hacker` '''
+def make_checked_in(modeladmin, request, queryset): # Needs to be Tested!!!
```

Comment:
I don't have much knowledge of the `admin` page for Django, so I might be missing some conventions, but `make_checked_in` is kinda unclear. To me, it implies that an object is being created.

`check_in` is a good alternative.